src/test/kotlin/com/atlassian/performance/tools/report/ShiftedDistributionRegressionTestTest.kt

```diff
@@ -174,7 +196,6 @@ class ShiftedDistributionRegressionTestTest {
         yAxisId = "latency-axis"
     )
 
-    @Ignore("Known bug: https://ecosystem.atlassian.net/browse/JPERF-1188")
```

Comment:
Mention the fix in changelog

Reply:
There was no fix. I just checked if the test passes, and it does